cmd/minikube/cmd/root.go

```diff
@@ -56,6 +56,7 @@ var RootCmd = &cobra.Command{
 			}
 		}
 
+		log.SetDebug(true)
```

Comment:
I think this might be slightly more clear (but have the same effect) if you do log.SetDebug(showLibmachineLogs) instead.